ratelimit/rate-limits.go

```diff
@@ -8,6 +8,42 @@ import (
 	"github.com/letsencrypt/boulder/strictyaml"
 )
 
+const (
+	// CertificatesPerName is the name of the CertificatesPerName rate limit
```

Comment:
I think a better way to write this to reduce the amount of duplicate text would be a comment for the whole const block something to the effect of
`// The ratelimits listed below allow mapping between ratelimit.Limits.<Name> to a string used by a prometheus metric`

Reply:
I'm not terribly concerned with repeating myself here and I believe it's generally good practice for each exported constant to have its own doc comment.